[2024-09-23T14:46:58.836+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-23T14:46:58.848+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 008-simple-extract-load-bq.row_quality_checks.check_row_data_1 manual__2024-09-23T11:32:25.296554+00:00 [queued]>
[2024-09-23T14:46:58.856+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 008-simple-extract-load-bq.row_quality_checks.check_row_data_1 manual__2024-09-23T11:32:25.296554+00:00 [queued]>
[2024-09-23T14:46:58.856+0000] {taskinstance.py:2865} INFO - Starting attempt 10 of 10
[2024-09-23T14:46:58.868+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCheckOperator): row_quality_checks.check_row_data_1> on 2024-09-23 11:32:25.296554+00:00
[2024-09-23T14:46:58.876+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2240) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-23T14:46:58.877+0000] {standard_task_runner.py:72} INFO - Started process 2248 to run task
[2024-09-23T14:46:58.878+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', '008-simple-extract-load-bq', 'row_quality_checks.check_row_data_1', 'manual__2024-09-23T11:32:25.296554+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/008-simple-extract-load-bq.py', '--cfg-path', '/tmp/tmp2omu4xl7']
[2024-09-23T14:46:58.879+0000] {standard_task_runner.py:105} INFO - Job 137: Subtask row_quality_checks.check_row_data_1
[2024-09-23T14:46:58.918+0000] {task_command.py:467} INFO - Running <TaskInstance: 008-simple-extract-load-bq.row_quality_checks.check_row_data_1 manual__2024-09-23T11:32:25.296554+00:00 [running]> on host 49bf9bae52f3
[2024-09-23T14:46:58.995+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='008-simple-extract-load-bq' AIRFLOW_CTX_TASK_ID='row_quality_checks.check_row_data_1' AIRFLOW_CTX_EXECUTION_DATE='2024-09-23T11:32:25.296554+00:00' AIRFLOW_CTX_TRY_NUMBER='10' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-23T11:32:25.296554+00:00'
[2024-09-23T14:46:58.998+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-09-23T14:46:58.998+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-09-23T14:46:58.999+0000] {logging_mixin.py:190} INFO - Current task name:row_quality_checks.check_row_data_1 state:running start_date:2024-09-23 14:46:58.850066+00:00
[2024-09-23T14:46:59.000+0000] {logging_mixin.py:190} INFO - Dag name:008-simple-extract-load-bq and current dag run status:running
[2024-09-23T14:46:59.000+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-23T14:46:59.000+0000] {baseoperator.py:405} WARNING - BigQueryCheckOperator.execute cannot be called outside TaskInstance!
[2024-09-23T14:46:59.001+0000] {sql.py:784} INFO - Executing SQL check: -- To zapytanie sprawdza, czy w tabeli znajdują się wiersze odpowiadające wartościom z walidacji.

SELECT
    COUNT(*) AS valid_rows  -- Liczba pasujących wierszy.
FROM
    `third-essence-345723.astronomer.forestfires`  -- Odwołanie do tabeli w BigQuery.
-- To zapytanie sprawdza, czy w tabeli znajdują się wiersze odpowiadające wartościom z walidacji.

SELECT
    COUNT(*) AS valid_rows  -- Liczba pasujących wierszy.
FROM
    `third-essence-345723.astronomer.forestfires`  -- Odwołanie do tabeli w BigQuery.
WHERE
    y = 1 AND  -- Sprawdzenie wartości y.
    month = 'jan' AND  -- Sprawdzenie wartości month (string).
    day = 'mon' AND  -- Sprawdzenie wartości day (string).
    ffmc = 85.4 AND  -- Sprawdzenie wartości ffmc.
    dmc = 26.2 AND  -- Sprawdzenie wartości dmc.
    dc = 94.3 AND  -- Sprawdzenie wartości dc.
    isi = 5.1 AND  -- Sprawdzenie wartości isi.
    temp = 22.1 AND  -- Sprawdzenie wartości temp.
    rh = 45 AND  -- Sprawdzenie wartości rh.
    wind = 4.5 AND  -- Sprawdzenie wartości wind.
    rain = 0.0 AND  -- Sprawdzenie wartości rain.
    area = 0.3  -- Sprawdzenie wartości area.
[2024-09-23T14:46:59.012+0000] {base.py:84} INFO - Retrieving connection 'bigquery'
[2024-09-23T14:46:59.063+0000] {sql.py:509} INFO - Running statement: -- To zapytanie sprawdza, czy w tabeli znajdują się wiersze odpowiadające wartościom z walidacji.

SELECT
    COUNT(*) AS valid_rows  -- Liczba pasujących wierszy.
FROM
    `third-essence-345723.astronomer.forestfires`  -- Odwołanie do tabeli w BigQuery.
-- To zapytanie sprawdza, czy w tabeli znajdują się wiersze odpowiadające wartościom z walidacji.

SELECT
    COUNT(*) AS valid_rows  -- Liczba pasujących wierszy.
FROM
    `third-essence-345723.astronomer.forestfires`  -- Odwołanie do tabeli w BigQuery.
WHERE
    y = 1 AND  -- Sprawdzenie wartości y.
    month = 'jan' AND  -- Sprawdzenie wartości month (string).
    day = 'mon' AND  -- Sprawdzenie wartości day (string).
    ffmc = 85.4 AND  -- Sprawdzenie wartości ffmc.
    dmc = 26.2 AND  -- Sprawdzenie wartości dmc.
    dc = 94.3 AND  -- Sprawdzenie wartości dc.
    isi = 5.1 AND  -- Sprawdzenie wartości isi.
    temp = 22.1 AND  -- Sprawdzenie wartości temp.
    rh = 45 AND  -- Sprawdzenie wartości rh.
    wind = 4.5 AND  -- Sprawdzenie wartości wind.
    rain = 0.0 AND  -- Sprawdzenie wartości rain.
    area = 0.3  -- Sprawdzenie wartości area., parameters: None
[2024-09-23T14:46:59.064+0000] {bigquery.py:1702} INFO - Inserting job ***_1727102819064006_a7be964720e57a73ad2ed9963c82cc4d
[2024-09-23T14:46:59.482+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 292, in execute
    super().execute(context=context)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py", line 785, in execute
    records = self.get_db_hook().get_first(self.sql, self.parameters)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 332, in get_first
    return self.run(sql=sql, parameters=parameters, handler=fetch_one_handler)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 456, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 514, in _run_command
    cur.execute(sql_statement)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2924, in execute
    job = self._run_query(sql)
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 3078, in _run_query
    job = self.hook.insert_job(configuration=configuration, project_id=self.project_id, location=location)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/google/common/hooks/base_google.py", line 559, in inner_wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1708, in insert_job
    job_api_repr.result(timeout=timeout, retry=retry)
  File "/home/airflow/.local/lib/python3.12/site-packages/google/cloud/bigquery/job/query.py", line 1590, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.12/site-packages/google/cloud/bigquery/job/query.py", line 1579, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.12/site-packages/google/cloud/bigquery/job/base.py", line 971, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/google/api_core/future/polling.py", line 261, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 9.1 - 25.14: Only one query can be executed at a time.; reason: invalidQuery, location: query, message: 9.1 - 25.14: Only one query can be executed at a time.

Location: US
Job ID: airflow_1727102819064006_a7be964720e57a73ad2ed9963c82cc4d

[2024-09-23T14:46:59.488+0000] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-09-23T14:46:59.488+0000] {logging_mixin.py:190} INFO - Task start:2024-09-23 14:46:58.850066+00:00 end:2024-09-23 14:46:59.488390+00:00 duration:0.638324
[2024-09-23T14:46:59.489+0000] {logging_mixin.py:190} INFO - Task:<Task(BigQueryCheckOperator): row_quality_checks.check_row_data_1> dag:<DAG: 008-simple-extract-load-bq> dagrun:<DagRun 008-simple-extract-load-bq @ 2024-09-23 11:32:25.296554+00:00: manual__2024-09-23T11:32:25.296554+00:00, state:running, queued_at: 2024-09-23 14:46:44.448246+00:00. externally triggered: True>
[2024-09-23T14:46:59.489+0000] {logging_mixin.py:190} INFO - Failure caused by 400 9.1 - 25.14: Only one query can be executed at a time.; reason: invalidQuery, location: query, message: 9.1 - 25.14: Only one query can be executed at a time.

Location: US
Job ID: ***_1727102819064006_a7be964720e57a73ad2ed9963c82cc4d
[2024-09-23T14:46:59.489+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=008-simple-extract-load-bq, task_id=row_quality_checks.check_row_data_1, run_id=manual__2024-09-23T11:32:25.296554+00:00, execution_date=20240923T113225, start_date=20240923T144658, end_date=20240923T144659
[2024-09-23T14:46:59.494+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-23T14:46:59.494+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 137 for task row_quality_checks.check_row_data_1 (400 9.1 - 25.14: Only one query can be executed at a time.; reason: invalidQuery, location: query, message: 9.1 - 25.14: Only one query can be executed at a time.

Location: US
Job ID: airflow_1727102819064006_a7be964720e57a73ad2ed9963c82cc4d
; 2248)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 292, in execute
    super().execute(context=context)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py", line 785, in execute
    records = self.get_db_hook().get_first(self.sql, self.parameters)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 332, in get_first
    return self.run(sql=sql, parameters=parameters, handler=fetch_one_handler)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 456, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 514, in _run_command
    cur.execute(sql_statement)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2924, in execute
    job = self._run_query(sql)
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 3078, in _run_query
    job = self.hook.insert_job(configuration=configuration, project_id=self.project_id, location=location)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/google/common/hooks/base_google.py", line 559, in inner_wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1708, in insert_job
    job_api_repr.result(timeout=timeout, retry=retry)
  File "/home/airflow/.local/lib/python3.12/site-packages/google/cloud/bigquery/job/query.py", line 1590, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.12/site-packages/google/cloud/bigquery/job/query.py", line 1579, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.12/site-packages/google/cloud/bigquery/job/base.py", line 971, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/google/api_core/future/polling.py", line 261, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 9.1 - 25.14: Only one query can be executed at a time.; reason: invalidQuery, location: query, message: 9.1 - 25.14: Only one query can be executed at a time.

Location: US
Job ID: airflow_1727102819064006_a7be964720e57a73ad2ed9963c82cc4d

[2024-09-23T14:46:59.515+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-09-23T14:46:59.528+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-23T14:46:59.530+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
